lib/publiccloud/instance.pm

```diff
@@ -379,6 +379,8 @@ sub wait_for_ssh {
     # Input parameters, see description in above head2 - Parameters section:
     $args{timeout} = get_var('PUBLIC_CLOUD_SSH_TIMEOUT', $args{timeout} // 600);
     $args{wait_stop} //= 0;
+    $args{scan_ssh_host_key} //= 0;
+    record_info('ZZZ', $args{scan_ssh_host_key});
```

Comment:
Please, give a valid name to the info box and all similar below, AAA, BBB, ... :smile:

Reply:
those will be gone.